docs/faq.rst

```diff
@@ -334,6 +334,145 @@ After successfull login the domain part will be striped and the rest used as use
+   
+.. image:: assets/screenshots/SpamFiltering.png
+
+The location in the administration web interface where the spam filter and spam filter tolerance can be configured.
+
+*Issue reference:* `1167`_.
+
+Can I learn ham/spam messages from an already existing mailbox?
+```````````````````````````````````````````````````````````````
+
+Mailu is supporting automatic spam learning for messages moved to the Junk mailbox. Any email moved from the Junk Folder will be learnt as ham. 
+
+If you already have an existing mailbox and want Mailu to learn them all as ham messages, you might run rspamc from within the dovecot container:
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 13 fuzzy_add /mail/user\@example.com/.Ham_Learn/cur/
+
+This should learn every file located in the ``Ham_Learn`` folder from user@example.com 
+
+Likewise, to lean all messages within the folder ``Spam_Learn`` as spam messages :
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 11 fuzzy_add /mail/user\@example.com/.Spam_Learn/cur/
+
+*Issue reference:* `1438`_.
+
+How can I block emails from a domain?
+`````````````````````````````````````
+
+Via the multimap filter it is possible to block emails from a sender domain. See the `official rspamd documentation`_ for more information. A local blacklist that contains the domains to be blocked can be configured via the Rspamd overrides folder. 
+
+The following steps have to be taken to configure an additional symbol (rule) that uses the multimap filter to block emails from sender domain.
+
+1. In the overrides folder create a configuration file for the multimap filter. This configuration is included by Rspamd in the main multimap configuration file. This means you do not have to use the "multimap {}" element. Files in the /mailu/overrides/rspamd/ folder folder are mapped to /etc/rspamd/override.d.
+   Create the file /mailu/overrides/rspamd/multimap.conf with contents:
+
+   .. code-block:: bash
+
+    #override.d/multimap.conf
+    #Tip: Each setting must be closed with a semi-colon ';'.
+    local_bl_domain {
+      type = "from";
+      filter = "email:domain";
+      map = "/etc/rspamd/override.d/blacklist.inc";
+      score = 14;
+      description = "Senders domain part is on the local blacklist";
+      group = "local_bl";
+      action = "reject";
+    }
+
+   Note the "action = "reject";" line. This is a so-called pre-filter. If you omit this line, then the configured score will be added to the total score of the email message. Depending on the end-score after processing all rules, a verdict is made. To override this, you can add the action line. When this symbol (rule) is fired, then this action is immediately taken and no further processing occurs. You can use the following actions:
+  
+   * discard: drop an email message, but return success for sender (should be used merely in special cases)
+
+   * reject: reject the email message
+
+   * add header: add email header to indicate spam
```

Comment:
How does this work? Does this enforce a header no matter what the final score is?
Or don't we get a header without this at all?
I thought that headers are added according to a final score after all rules.

Reply:
Yes. These are pre-filters. This will add an header no matter what the final score is. Using a pre-filter also means that no further rules are processed. 

This is already explained in the paragraph before this list:
_Note the "action = "reject";" line. This is a so-called pre-filter. If you omit this line, then the configured score will be added to the total score of the email message. Depending on the end-score after processing all rules, a verdict is made. To override this, you can add the action line. When this symbol (rule) is fired, then this action is **immediately taken and no further processing occurs.** You can use the following actions:_